src/Grid/Column/Column.php

```diff
@@ -14,40 +14,35 @@
 
 use APY\DataGridBundle\Grid\Filter;
 use APY\DataGridBundle\Grid\Row;
-use Doctrine\Common\Version as DoctrineVersion;
+use Doctrine\ORM\Version;
```

Comment:
@cpkdevries looks like this one was missed. At least a response is missing.

Reply:
Sorry - just didn't comment. We discussed - this was only being used for the constant in the version class here. Because it related to a very old version or doctrine/orm, I added a conflict to `composer.json` instead to deal with this.

Reply:
Update - see [latest comment.](https://github.com/northernco/NorthernDatagridBundle/pull/22#issuecomment-1497512277)